website/source/guides/acl.html.markdown

```diff
@@ -0,0 +1,309 @@
+    https://nomad.rocks/v1/acl/policy/anonymous
+
+# Verify anonymous request works
+$ curl https://nomad.rocks/v1/jobs
+```
+
+# Rule Specification
+
+A core part of the ACL system is the rule language which is used to describe the policy that must be enforced.
+We make use of the [HashiCorp Configuration Language (HCL)](https://github.com/hashicorp/hcl/) to specify rules.
+This language is human readable and interoperable with JSON making it easy to machine-generate. Policies can contain any number of rules.
+
+Policies typically have several dispositions:
+
+* `read`: allow the resource to be read but not modified
+* `write`: allow the resource to be read and modified
+* `deny`: do not allow the resource to be read or modified. Deny takes precedence when multiple policies are associated with a token.
+
+Specification in the HCL format looks like:
+
+```text
+# Allow read only access to the default namespace
+namespace "default" {
+    policy = "read"
+}
+
+# Allow writing to the `foo` namespace
+namespace "foo" {
+    policy = "write"
+}
+
+agent {
+    policy = "read"
+}
+node {
+    policy = "read"
+}
+```
+
+This is equivalent to the following JSON input:
+
+```json
+{
+    "namespace": {
+        "default": {
+            "policy": "read"
+        },
+        "foo": {
+            "policy": "write"
+        }
+    },
+    "agent": {
+        "policy": "read"
+    },
+    "node": {
+        "policy": "read"
+    }
+}
+```
+
+The [ACL Policy](/api/acl-policies.html) API allows either HCL or JSON to be used to define the content of the rules section.
+
+### Namespace Rules
+
+The `namespace` policy controls access to a namespace, including the [Jobs API](/api/jobs.html), [Deployments API](/api/deployments.html), [Allocations API](/api/allocations.html), and [Evaluations API](/api/evaluations.html).
+
+```
+namespace "default" {
+    policy = "write"
+}
+namespace "sensitive" {
+    policy = "read"
+}
+```
+
+Namespace rules are keyed by the namespace name they apply to. When no namespace is specified, the "default" namespace is the one used. For example, the above policy grants writeaccess to the default namespace, and read access to the sensitive namespace. In addition to the coarse grained `policy` specification, the `namespace` stanza allows setting a more fine grained list of `capabilities`. This includes:
+
+* `deny` - When multiple policies are associated with a token, deny will take precedence and prevent any capabilities.
+* `list-jobs` - Allows listing the jobs and seeing coarse grain status.
+* `read-job` - Allows inspecting a job and seeing fine grain status.
+* `submit-job` - Allows jobs to be submitted or modified.
+* `read-logs` - Allows the logs associated with a job to be viewed.
+* `read-fs` - Allows the filesystem of allocations associated to be viewed.
+
+The coarse grained policy dispositions are shorthand for the fine grained capabilities:
+
+* `deny` policy - ["deny"]
+* `read` policy - ["list-jobs", "read-jobs"]
```

Comment:
They are too sensitive, since most apps are bad and end up logging sensitive information. Keeping to principle of least surprise, if you grant an operator the ability to read jobs status information, it's weird if they could potentially extract sensitive information.